backend/src/controllers/report.js

```diff
@@ -64,6 +64,90 @@ const getOne = async (request, reply) => {
   }
 };
 
+/**
+ * Get a specific item by ID.
+ *
+ * @param   {FastifyRequest} request FastifyRequest is an instance of the standard http or http2 request objects.
+ * @param   {FastifyReply}   reply   FastifyReply is an instance of the standard http or http2 reply types.
+ * @returns {object}
+ */
+const getProjectBudgetReport = async (request, reply) => {
+  if (userCan(request, "report_read_mine") && checkMine(request)) {
```

Comment:
don't think this would be the correct permission, i think it should just be
```suggestion
  if (userCan(request, "report_read_all")) {
```
we will have to look if this is true, but the read_mine && checkMIne should not be used.

Reply:
this will be ok for now, this will be fixed this coming sprint, when we review caps and roles and controllers